apps/root/src/frame/components/wallet-menu/index.tsx

```diff
@@ -136,12 +204,119 @@ const WalletMenu = ({ open, onClose }: WalletMenuProps) => {
           </StyledRecentTransactionsTitleContainer>
           <StyledAccount>
             <Typography variant="subtitle1" fontWeight={500}>
-              <Address address={account} trimAddress trimSize={10} />
+              <Address address={account || ''} trimAddress trimSize={10} />
             </Typography>
           </StyledAccount>
+          {user?.type === UserType.privy && (
+            <>
+              <Typography variant="body2" component="span">
+                <FormattedMessage description="connect another account" defaultMessage="Connect another account:" />
+              </Typography>
+              <StyledLoginOptionsContainer>
+                <Button
+                  variant="outlined"
+                  color="default"
+                  disabled={!!user.privyUser?.discord}
+                  onClick={linkDiscord}
+                  endIcon={user.privyUser?.discord ? <StyledCheckCircleOutlineIcon /> : null}
+                >
+                  <DiscordIcon size="24px" />
+                  {user.privyUser?.discord?.username}
+                </Button>
+                <Button
+                  variant="outlined"
+                  color="default"
+                  disabled={!!user.privyUser?.email}
+                  onClick={linkEmail}
+                  endIcon={user.privyUser?.email ? <StyledCheckCircleOutlineIcon /> : null}
+                >
+                  <EmailIcon />
+                  {user.privyUser?.email?.address}
+                </Button>
+                <Button
+                  variant="outlined"
+                  color="default"
+                  disabled={!!user.privyUser?.twitter}
+                  onClick={linkTwitter}
+                  endIcon={user.privyUser?.twitter ? <StyledCheckCircleOutlineIcon /> : null}
+                >
+                  <TwitterIcon />
+                  {user.privyUser?.twitter?.username}
+                </Button>
+                <Button variant="outlined" color="default" onClick={linkWallet}>
+                  <WalletIcon />
+                </Button>
+                <Button
+                  variant="outlined"
+                  color="default"
+                  disabled={!!user.privyUser?.google}
+                  onClick={linkGoogle}
+                  endIcon={user.privyUser?.google ? <StyledCheckCircleOutlineIcon /> : null}
+                >
+                  <GoogleIcon />
+                  {user.privyUser?.google?.name}({user.privyUser?.google?.email})
```

Comment:
This shows empty parenthesis if no google account is linked
<img width="77" alt="Screenshot 2023-10-17 at 12 00 47" src="https://github.com/Mean-Finance/dca-fe/assets/144699868/c1076cf5-f90d-40d3-98f7-98b68e874c73">